dvc/tree/gdrive.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, repo, config):
             )
 
         self._bucket = self.path_info.bucket
+        self._path = self.path_info.path
```

Comment:
This one doesn't belong here, but gdrive is the only implementation that is trying to use it. We've ran into it during `import-url` work, will send a separate PR for that.